lib/views/staging-view.js

```diff
@@ -797,6 +846,11 @@ export default class StagingView extends React.Component {
       return;
     }
 
+    addEvent('undo-last-discard', {
```

Comment:
same comment as above - would recommend refactoring this so that you're logging menu views as a ui interaction.